### DISCUSSION TITLE
Is there an autocomplete plugin for tabby?

### DISCUSSION THREAD

**Discussion by 0xrushi** [Eugeny/tabby#6332] [Q&A]
Hi all,

I just came across tabby, Since this is made using electron, I was wondering if there is some plugin for auto completion of commands.

**Comment by lin-calvin** [comment]
you should findr a shell plugin instead of a tabby plugin

**Comment by Eugeny** [comment]
Autocompletion is provided by the shell, not by a terminal it's running in.

**Reply by seanwwh** [reply]
If Tabby support autocompletion in terminal, just like WindTerm(https://kingtoolbox.github.io/2022/05/18/auto_completion/), I think it will be more friendly for users, I hope this feature will be considered again, thank you.

**Comment by 0xrushi** [comment]
@calvinweb @Eugeny something like this, found in IDEs

![image](https://user-images.githubusercontent.com/6279035/166264732-2f4c9800-638d-40f4-a796-c6270aa1fb1d.png)

**Reply by Eugeny** [reply]
Try Fig (https://fig.io/), which has Tabby support

**Reply by 0xrushi** [reply]
I came here from fig. Fig is still heavily under development for macos. For other platforms it'll be released next year I guess.

**Reply by gitthangbaby** [reply]
Fig is gone now. Focus shifted to Warp.

**Comment by JackBailey** [comment]
Any chance for one for passwords (which would be Tabby-side) like Termius has? 
![termius password login prompt](https://cdn.productboard.com/production/attachments/e36306d67928c5d501b953d68ee61e88a26d11ef6660d747f33fbf5ae0af50cf/portal_cover/password_paste.png)
